backend/census_historical_migration/workbooklib/end_to_end_core.py

```diff
@@ -250,10 +251,11 @@ def api_check(json_test_tables):
     return combined_summary
 
 
-def run_end_to_end(user, audit_header, result):
+def run_end_to_end(user, audit_header):
     """Attempts to migrate the given audit"""
     try:
-        sac, result = setup_sac(user, audit_header, result)
+        migration_result.result = migration_result.DEFAULT_RESULT
```

Comment:
Does this work ? I would expect these two objects to point to the same memory location due to `result = DEFAULT_RESULT` in migration_result.py and therefore here instead doing a reset it just reaffirms that `migration_result.result` is pointing to the now-modified `DEFAULT_RESULT`

Reply:
Yep, now using copy.deepcopy()